webapp/src/main/java/ar/edu/itba/paw/webapp/controllers/old/OldStudentController.java

```diff
@@ -0,0 +1,783 @@
+package ar.edu.itba.paw.webapp.controllers.old;
+
+import ar.edu.itba.paw.interfaces.StudentService;
+import ar.edu.itba.paw.models.Course;
+import ar.edu.itba.paw.models.Grade;
+import ar.edu.itba.paw.models.users.Student;
+import ar.edu.itba.paw.shared.CourseFilter;
+import ar.edu.itba.paw.shared.StudentFilter;
+import ar.edu.itba.paw.webapp.auth.UserSessionDetails;
+import ar.edu.itba.paw.webapp.forms.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.MessageSource;
+import org.springframework.security.core.Authentication;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.servlet.ModelAndView;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.validation.Valid;
+import java.util.List;
+import java.util.Locale;
+
+//@Controller
+public class OldStudentController {
+
+	private final static Logger LOGGER = LoggerFactory.getLogger(OldStudentController.class);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AModifiersOrderCheck)